exp-models/addon/models/experiment.js

```diff
@@ -122,6 +122,10 @@ export default DS.Model.extend(JamModel, {
         // Eg an experiment called 'test0' would have a collection 'session-test0'
         return `session${this.get('shortId')}s`; // FIXME: the spurious s is a requirement imposed by genschemas...
     }),
+    // Only use to check that the session collection is created when an
+    // experiment document is created. NOT a reliable way to fetch an
+    // experiment's session collection
+    _sessionCollection: null,
```

Comment:
I'm not trying to encourage people to use this as a convenient way to fetch the experiment's sessions collection. This approach, while limited requires much less logic.